cachi2/core/extras/envfile.py

```diff
@@ -24,7 +31,14 @@ def based_on_suffix(cls, filepath: Path):
             reason = (
                 f"file has no suffix: {filepath}" if not suffix else f"unsupported suffix: {suffix}"
             )
-            raise UnsupportedFeature(f"Cannot determine envfile format, {reason}") from e
+            raise UnsupportedFeature(
+                f"Cannot determine envfile format, {reason}",
+                solution=(
+                    f"Please use one of the supported suffixes: {cls._suffixes_repr()}\n"
+                    "You can also define the format explicitly instead of letting Cachi2 choose.\n"
+                    "If you need Cachi2 to support a new format, please contact the maintainers."
```

Comment:
Should we even suggest them to ask us for a new format? :smile: 
I'd probably just drop this sentence here.

Reply:
True, let's not give them any ideas :smile: